sites/bizbash.com/config/navigation.js

```diff
@@ -7,11 +7,12 @@ const topics = [
 ];
 
 const secondaryItems = [
-  { href: '/webinars', label: 'Webinars' },
+  { href: '/webinars', label: 'On-Demand' },
```

Comment:
On-Demand will go to the new on-demand section page so change the alias here to /on-demand